src/parser.js

```diff
@@ -38,7 +38,7 @@ function resolveParseFunction(opts) {
       return eval("require")(path.resolve(process.cwd(), opts.parser));
     } catch (err) {
       /* istanbul ignore next */
-      throw new Error(`Couldn't resolve parser "${opts.parser}"`);
+      throw `Couldn't resolve parser "${opts.parser}"`;
```

Comment:
This works, but is it the correct fix? Do we want to catch errors somewhere instead?